diff -aur a/config.h b/config.h
--- a/config.h	2016-10-31 09:48:24.000000000 -0400
+++ b/config.h	2017-11-16 16:02:28.680969571 -0500
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Fixed:pixelsize=18:antialias=false:autohint=false";
+static char font[] = "xos4 Terminus:pixelsize=14";
 static int borderpx = 2;
 
 /*
@@ -63,7 +63,7 @@
 static int bellvolume = 0;
 
 /* default TERM value */
-static char termname[] = "st-256color";
+static char termname[] = "xterm-256color";
 
 /*
  * spaces per tab
